build.sh

```diff
@@ -2,53 +2,200 @@
 
 set -Eeou pipefail
 
-# color codes
+# fold me :) (in vscode vscodium etc)
+#region DEFAULTS_AND_COMMAND_LINE_ARGUMENTS
+# Parse command-line arguments for flags
+verbose=false
+UBPORTSDOCSENV=${UBPORTSDOCSENV:-"$HOME/ubportsdocsenv"}
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    --verbose)
+      verbose=true
+      shift
+      ;;
+    --env)
+      # If --env flag is passed, set UBPORTSDOCSENV to the next argument value
+      UBPORTSDOCSENV=${2-$UBPORTSDOCSENV}
+      shift 2
+      ;;
+    *)
+      shift
```

Comment:
Let's be a little more defensive here and error out instead of silently ignoring invalid options